README.md

```diff
@@ -2,6 +2,8 @@
 
 This repo contains the pairing test exercises that are used in the Guardian recruitment process.
 
+Interviewers, please read the [guidelines](https://docs.google.com/document/d/1DegP-3YBTzJoM3tLuo-yenZjDqfFRTOW2zozXBzdnTE)
+
 [Why have we published our exercises?](https://www.theguardian.com/info/developer-blog/2016/jan/20/the-guardians-new-pairing-exercises)
 
 Inspired to [work for us](http://developers.theguardian.com/join-the-team.html)?
```

Comment:
Because I think they are more important for people coming to see what our exercises are about.